### PR TITLE
docs: 📚 add migration guide for sd-video

### LINKED ISSUE #602: docs: 📚 add migration guide for `sd-video`
## Description
The `sd-video` is missing a migration guide from the respective ui-component (Component Library) to our SDS Component.

A migration guide template can be found in the root folder of the repository, under: `templates/migration-guide-template.mdx`

Link to the respective ui-component: https://dev.azure.com/Union-Investment/Frontend/_git/union-investment-frontend?path=/packages/libs/component-library/src/components/ui-video
## Where to put the documentation
Storybook
## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Migration Guide has been created/updated (if applicable)

### PR DESCRIPTION
**Pull request by Vahid1919**
## Description:
Added migration guide for sd-video. Closes #602.

## Definition of Reviewable:
*PR notes: Irrelevant elements should be removed.*
- [x] Migration Guide is created/updated
- [x] relevant tickets are linked
- [x] PR is assigned to project board

### CODE REVIEW

**Review by yoezlem — APPROVED** [review]

File: packages/components/src/docs/Migration/ui-video.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Migration Guide: From `ui-video` to `sd-video`
```
Comment (karlbaumhauer) [inline packages/components/src/docs/Migration/ui-video.mdx:1]: I would love to see a more detailed description about the sd-video component only provided a wrapper compared to the ui-video providing the whole player. Its clear if you know the component but would be nice to have it clear mentioned in the beginning of the component.

**Review by karlbaumhauer — CHANGES_REQUESTED** [review]

**Review by karlbaumhauer — APPROVED** [review]
LGTM thx

**Review by abudd1094 — APPROVED** [review]
Thanks for adding this! I overlooked it when creating.

File: packages/components/src/docs/Migration/ui-video.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## ⚙️ Attributes
+
+### ✨ New Attributes
+
+#### [playing]
+
+Setting this to `true` will hide the play icon and the overlay.
+
+#### [isBelowBreakpoint]
```
Comment (mariohamann) [inline packages/components/src/docs/Migration/ui-video.mdx:31]: Is this a public attribute?

**Review by azraefendic — APPROVED** [review]

File: packages/components/src/docs/Migration/ui-video.mdx (hunk 2 above)
Comment (Vahid1919) [reply packages/components/src/docs/Migration/ui-video.mdx:31]: Yup!
image

**Pull request merged**